application/core/plugins/AuthLDAP/AuthLDAP.php

```diff
@@ -230,8 +230,8 @@ private function _createNewUser($new_user, $password = null)
         if (!is_resource($ldapconn)) {
```

Comment:
The issue seems more here :)

Reply:
$ldacon return ressource in 7.X, Class in 8.1 and array (with error) on both.

You can just replace here by the same test `if (isset($ldapconn['errorCode'])) {`